features/machine/autoscaler.feature

```diff
@@ -7,14 +7,14 @@ Feature: Cluster Autoscaler Tests
   Scenario: Cluster should automatically scale up and scale down with clusterautoscaler deployed
     Given I have an IPI deployment
     And I switch to cluster admin pseudo user
+    And I use the "openshift-machine-api" project
 
     Given I store the number of machines in the :num_to_restore clipboard
-    And admin ensures node number is restored to "<%= cb.num_to_restore %>" after scenario
+    And admin ensures machine number is restored to "<%= cb.num_to_restore %>" after scenario
```

Comment:
Does  "admin ensures machine number is " to "admin ensures total machines is"  looks good ?

Reply:
Hi @miyadav I think the current step looks already clear and straightforward.